pom.xml

```diff
@@ -97,6 +97,7 @@
 
   <modules>
     <module>core</module>
+    <module>platform</module>
```

Comment:
What was the reasoning behind creating a new top-level module? Aren't these commands behaving as another type of Northbound provider?

Reply:
> I agree that for this first release, it only interacts with northbound providers.

I would go a little further and say that it **is** a northbound provider, in that it is using the mechanisms other northbound providers do to provide access to data from the sensiNact core.

> Do you see this as eventually expanding to interface with southbound providers as well?

I think that if we create commands which are doing the job of a southbound provider I would expect them to be compiled and packaged separately, and that those commands would live in the `southbound` module.

If we start having commands that do other things (not that I can think what those might be at this time) then at that point I can see that we might start a separate top level module for the build.

Reply:
Ok. In that case, I'll remove the `platform` package and find a different home. Any preferences?

Reply:
Definitely in the `northbound` folder